tests/instrument_controllers/qdevil/test_qdevil_qdac2_controller.py

```diff
@@ -0,0 +1,59 @@
+import copy
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from qililab.instrument_controllers.qdevil.qdevil_qdac2_controller import QDevilQDac2Controller
+from qililab.instruments.qdevil.qdevil_qdac2 import QDevilQDac2
+from qililab.platform import Platform
+from qililab.settings import Settings
+from tests.data import SauronQDevil  # pylint: disable=import-error
+from tests.test_utils import build_platform  # pylint: disable=import-error
+
+
+@pytest.fixture(name="platform")
+def fixture_platform() -> Platform:
+    """Return Platform object."""
+    return build_platform(runcard=SauronQDevil.runcard)
```

Comment:
perhaps it'd be better to have the runcard being mocked elsewhere (check https://qilimanjaro.slack.com/archives/C05D7KW4B6G/p1702912592701119)